client/state/login/magic-login/actions.js

```diff
@@ -13,6 +22,8 @@ import {
 	MAGIC_LOGIN_SHOW_CHECK_YOUR_EMAIL_PAGE,
 	MAGIC_LOGIN_SHOW_REQUEST_FORM,
 } from 'state/action-types';
+
+import config from 'config';
 import wpcom from 'lib/wp';
```

Comment:
Minor: I would place the default exports `config` and `wpcom` above the named ones.